android/src/main/java/io/radar/react/RNRadarModule.java

```diff
@@ -87,16 +87,24 @@ public void getPermissionsStatus(Promise promise) {
             return;
         }
 
-        boolean foreground = ActivityCompat.checkSelfPermission(getReactApplicationContext(), Manifest.permission.ACCESS_FINE_LOCATION) == PackageManager.PERMISSION_GRANTED;
+        Activity activity = getCurrentActivity();
+
+        boolean foreground = ActivityCompat.checkSelfPermission(activity, Manifest.permission.ACCESS_FINE_LOCATION) == PackageManager.PERMISSION_GRANTED;
+        boolean background = true;
+        boolean denied = ActivityCompat.shouldShowRequestPermissionRationale(activity, Manifest.permission.ACCESS_FINE_LOCATION);
```

Comment:
`shouldShowRequestPermissionRationale()` returns `true` if a permission prompt has been previously shown and permissions were denied: https://developer.android.com/training/permissions/requesting#explain